blocks/library/block/index.js

```diff
@@ -172,20 +172,10 @@ export const name = 'core/block';
 
 export const settings = {
 	title: __( 'Shared Block' ),
-	category: 'shared',
-	isPrivate: true,
-
-	attributes: {
-		ref: {
-			type: 'number',
-		},
-	},
 
-	supports: {
-		customClassName: false,
-		html: false,
-	},
+	isPrivate: true,
```

Comment:
Why is this on the client?

Reply:
I don't like this property at all :) I will move it to the server, too 👍

Reply:
To make things easier, I moved `isPrivate` to `supports` property and renamed to `insertable`.